src/openvic-simulation/dataloader/ModManager.cpp

```diff
@@ -21,27 +22,40 @@ bool ModManager::load_mod_file(ast::NodeCPtr root) {
 	memory::vector<memory::string> replace_paths;
 	memory::vector<memory::string> dependencies;
 
-	bool ret = NodeTools::expect_dictionary_keys(
+	bool ret = NodeTools::expect_dictionary_keys_and_default(
+		key_value_success_callback,
```

Comment:
```suggestion
		map_key_value_ignore_invalid_callback<template_key_map_t<StringMapCaseSensitive>>,
```
`key_value_success_callback` accepts anything.
`map_key_value_ignore_invalid_callback` warns when finding invalid keys and lists valid keys.
That's very useful when dealing with typo's and such.